darts/models/forecasting/ensemble_model.py

```diff
@@ -377,6 +386,117 @@ def _predictions_reduction(
+            ..
+
+        Parameters
+        ----------
+        path
+            Path or file handle under which to save the ensemble model at its current state. If no path is specified,
+            the ensemble model is automatically saved under ``"{RegressionEnsembleModel}_{YYYY-mm-dd_HH_MM_SS}.pkl"``.
+            If the ith model of `forecasting_models` is a TorchForecastingModel, two files (model object and checkpoint)
+            would be saved under ``"{path}.{ithModelClass}_{i}.pt"`` and ``"{path}.{ithModelClass}_{i}.ckpt"``
+            E.g., ``"RegressionEnsembleModel_2024-07-25_14_58_53.pkl"``
+            ``"RegressionEnsembleModel_2024-07-25_14_58_53.pkl.TiDEModel_1.pt"``
+            ``"RegressionEnsembleModel_2024-07-25_14_58_53.pkl.TiDEModel_1.ckpt"``
+        pkl_kwargs
+            Keyword arguments passed to `pickle.dump()`
+        """
+
+        if path is None:
+            # default path
+            path = self._default_save_path() + ".pkl"
+
+        if isinstance(path, (str, os.PathLike)):
+            # save the whole object using pickle
+            with open(path, "wb") as handle:
+                pickle.dump(obj=self, file=handle, **pkl_kwargs)
+        elif isinstance(path, io.BufferedWriter):
+            # save the whole object using pickle
+            pickle.dump(obj=self, file=path, **pkl_kwargs)
+        else:
+            raise_log(
+                ValueError(
+                    "Argument 'path' has to be either 'str' or 'PathLike' (for a filepath) "
+                    f"or 'BufferedWriter' (for an already opened file), but was '{path.__class__}'."
+                ),
+                logger=logger,
+            )
+
+        for i, m in enumerate(self.forecasting_models):
+            if TORCH_AVAILABLE and issubclass(type(m), TorchForecastingModel):
+                path_pt_ckpt = f"{path}.{type(m).__name__}_{i}.pt"
+                m.save(path=path_pt_ckpt)
+
+    @staticmethod
+    def load(path: Union[str, os.PathLike, BinaryIO]) -> "EnsembleModel":
+        """
+        Loads the ensemble model from a given path or file handle.
+
+        Parameters
+        ----------
+        path
+            Path or file handle from which to load the ensemble model.
+        """
+
+        if isinstance(path, (str, os.PathLike)):
+            raise_if_not(
+                os.path.exists(path),
+                f"The file {path} doesn't exist",
+                logger,
+            )
+
+            with open(path, "rb") as handle:
+                model = pickle.load(file=handle)
+        elif isinstance(path, io.BufferedReader):
+            model = pickle.load(file=path)
+        else:
+            raise_log(
+                ValueError(
+                    "Argument 'path' has to be either 'str' or 'PathLike' (for a filepath) "
+                    f"or 'BufferedReader' (for an already opened file), but was '{path.__class__}'."
+                ),
+                logger=logger,
+            )
```

Comment:
we can use the existing load logic:

```suggestion
        model: EnsembleModel = GlobalForecastingModel.load(path)
```